Makefile

```diff
@@ -18,26 +43,16 @@ bucket:
 	           minio/mc \
 	           /bin/sh -c "chmod +x ./docker/minio/create-bucket.sh && ./docker/minio/create-bucket.sh"
 
-test:
-	docker-compose run composer ./vendor/bin/simple-phpunit
+##
+## Tests
+## -----
+##
 
-logs:
-	docker-compose logs -ft
+test: ## Run unit tests
+	docker-compose run php bin/phpunit
 
-bash:
-	docker-compose up -d php
-	docker-compose exec php bash
-
-.env: .env.dist
-	@if [ -f .env ]; then \
-		echo '\033[1;41mYour .env file may be outdated because .env.dist has changed.\033[0m';\
-		echo '\033[1;41mCheck your .env file, or run this command again to ignore.\033[0m';\
-		touch .env;\
-		exit 1;\
-	else\
-		echo cp .env.dist .env;\
-		cp .env.dist .env;\
-	fi
+behat: ## Run functional tests
```

Comment:
Add here in the help section that for Behat test to work, the containers must be up, so the command `initialize` must be launched before

Reply:
i thk you dont execute the command perhaps *run* do something dont need initialize

Reply:
What I'm saying is if you don't start the containers and just launch `make behat`, all the tests will fail because there are no user accounts, each test will give you:
```
Given I am logged in as an admin                                     # FeatureContext::iAmLoggedInAsAnAdmin()
      cURL error 6: Could not resolve host: nginx (see http://curl.haxx.se/libcurl/c/libcurl-errors.html) (GuzzleHttp\Exception\ConnectException)
```

Reply:
so php depend to nginx
my command is not the problem